README.md

```diff
@@ -85,6 +85,7 @@ Currently supported engines are:
 * `docker` - Builds in a docker container
 * `pooler` - Selects a vm from Puppet Labs' vm pooler to build on
 * `hardware` - Build on a specific taget and lock it in redis
+* `amazon` - Build on a specific AWS instance.
```

Comment:
I would like to just keep it as aws. But I would think ec2 is probably the only resource that vanagon would probably use.